frontend/ui/widget/configdialog.lua

```diff
@@ -591,20 +610,28 @@ function ConfigOption:init()
                     num_buttons = #self.options[c].values,
                     position = self.options[c].default_pos,
                     callback = function(arg)
+
                         if arg == "-" or arg == "+" then
+                            Notification:setNotifySource(Notification.SOURCE_BOTTOM_MENU_FINE)
                             self.config:onConfigFineTuneChoose(self.options[c].values, self.options[c].name,
                                 self.options[c].event, self.options[c].args, self.options[c].events, arg, self.options[c].hide_on_apply,
                                 self.options[c].fine_tune_param)
                         elseif arg == "⋮" then
+                            Notification:setNotifySource(Notification.SOURCE_BOTTOM_MENU_MORE)
                             self.config:onConfigMoreChoose(self.options[c].values, self.options[c].name,
                                 self.options[c].event, arg, name_text, self.options[c].more_options_param)
                         else
+                            Notification:setNotifySource(Notification.SOURCE_BOTTOM_MENU_TOGGLE)
```

Comment:
This if for ButtonProgressWidget.
I wonder if it would not need a specific SOURCE_BOTTOM_MENU_BUTTONS, because, unlike _TOGGLE where you see some text on the button you tap, with a ButtonProgressWidget, you don't see it, so it might be nice to still show a notification (mainly: `Line spacing 85%` and `Contrast set to 4`).
We could enable notifications for these for SOURCE_DEFAULT (and even for SOURCE_SOME ?).

Reply:
Shouldn't we call the `progress buttons` as `progress sliders` in the menu?
In my feeling the linespacing and contrast settings are not buttons but more like a slider, controller ...

Reply:
> as progress sliders in the menu?

Fine with me (as long as the internal name stays "progress buttons" as the widget internal name).

> I would prefer popups on a progress button only to be in default.

Fine with me too.
(Althought DEFAULT only adds More "...", where we usually see the value we are applying - so I would have not chose this one - but as I use that rarely, I'll be fine choosing DEFAULT and see a notif also on these number pickers.)